iina/VideoView.swift

```diff
@@ -238,36 +240,133 @@ class VideoView: NSView {
       actualFps = 60;
     }
     player.mpv.setDouble(MPVOption.Video.overrideDisplayFps, actualFps)
-    
-    setICCProfile(displayId)
-    currentDisplay = displayId
+
+    if #available(macOS 10.15, *) {
+      refreshEdrMode()
+    } else {
+      setICCProfile(displayId)
+    }
```

Comment:
I agree that this is the appropriate place to refresh EDR mode. However there's another case: on some monitors user can enable/disable HDR in system preferences. So we also need to add a listener in `windowDidLoad()` for `PlayerWindowController`.

```swift
addObserver(to: .default, forName: NSScreen.colorSpaceDidChangeNotification, object: nil) { [unowned self] notification in
  guard let screen = notification.object as? NSScreen { ... }
}
```

If you don't have the equipment for testing, please skip this and I can add it later.

Reply:
I will skip this